event-log-service-webapp/src/main/resources/sql-scripts/copyOfflineConcepts.py

```diff
@@ -5,19 +5,17 @@
 import getpass
 
 
-mySqlPswd = os.environ.get('MYSQL_ROOT_PASSWORD');
+mySqlUser = os.environ.get('MYSQL_USER');
+mySqlPswd = os.environ.get('MYSQL_PASSWORD');
 
-if 'None' == str(mySqlPswd) or '' == str(mySqlPswd):
-    try:
-       mySqlUser = raw_input('MySQL user: ')
-       mySqlPswd = getpass.getpass(prompt='MySQL password: ')
-    except Exception as error:
-       print 'ERROR: '+ error;
-else:
-    mySqlUser = 'root';
+if 'None' == str(mySqlUser) or '' == str(mySqlUser):
+    mySqlUser = raw_input('MySQL user: ')
```

Comment:
1) does other users apart from root users has permissions to connect to opemrs db and add offlinr concepts?
 if not then we don't need to take user as input, and always prompt for root user password.